content/300-guides/050-database/100-developing-with-prisma-migrate/050-add-prisma-migrate-to-a-project.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: Adding Prisma Migrate to an existing project
+metaDescription: Learn how to add Prisma Migrate to an existing project.
+tocDepth: 2
+---
+
+<TopBlock>
+
+This guide describes how to add [Prisma Migrate](../../../concepts/components/prisma-migrate) to an existing project.
+
+</TopBlock>
+
+## Update the development environment
+
+As part of adding Prisma Migrate to your development environment, you must **reset your development database**. This will result in data loss in the development database only.
+
+Production databases and any other database that cannot be reset should be [baselined to avoid data loss](#baseline-your-production-environment).
+
+### Introspect to create or update your Prisma schema
+
+Make sure your Prisma schema is in sync with your database schema. This should already be true if you are using a previous version of Prisma Migrate.
+
+1. Introspect the database to make sure that your Prisma schema is up-to-date:
+
+   ```terminal
+   prisma introspect
+   ```
+
+### Initialize migration history
+
+To initialize a new migration history:
+
+1. If you have a `prisma/migrations` folder, delete, move, rename, or archive this folder.
+
+1. Run the following command to create the first migration **without applying it**, in case you need to [modify the initial migration](#modify-initial-migration-to-include-feautres-not-supported-by-the-prisma-schema-language):
```

Comment:
The anchor in this link has a typo - either in the link or on the target page for real?